src/vsr.zig

```diff
@@ -61,13 +61,25 @@ pub const Zone = enum {
     wal_headers,
     wal_prepares,
     client_replies,
+    // Note: there might be padding between `client_replies` and `grid`, to make sure grid blocks
+    // are aligned to block size and not just to sector size. Aligning blocks this way makes it more
+    // likely that they are aligned to the underlying physical sector size.
```

Comment:
Originally I went with just adding an explicit `.gird_padding` Zone, but but that would require some non-sensial assertions that we _dont_ actually use this zone.